policies/glossary.md

```diff
@@ -0,0 +1,153 @@
+# Glossary of OpenSSL terms
+
+## ABI
+
+_Application binary interface_
+
+An ABI compatible release allows a shared library to be replaced with a
+new version or for applications to be relinked against the new library
+with the expectation that everything will run.
+
+ABI compatible releases are also [API](#api) compatible.
+
+## Alpha release
+
+An [alpha release] is an early pre-release version.
+Refer to the [versioning policy] for specific details.
+
+## API
+
+_Application programming interface_
+
+An API compatible release, that is not [ABI](#abi) compatible, allows
+application to be recompiled against the new library with the expectation
+that everything will run.
+
+Refer to the [API compatibility policy] for specific details.
+
+## Beta release
+
+A [beta release] is a late pre-release version.
+Refer to the [versioning policy] for specific details.
+
+## Bug fix
+
+A bug fix is a fix of functionality of the libraries, modules, applications
+or the build system.
+Refer to the [stable release updates policy] for specific details.
+
+## CCLA
+
+_Corporate Contributor Licence Agreement_
+
+The [CCLA] is a legal document that grants certain rights to code
+contributed as part of the employment of the contributor.  All non-trivial
+contributions on behalf of an employer must be submitted under a CCLA.
+
+Also see: [ICLA](#icla)
+
+## CHANGES
+
+Significant modifications for the OpenSSL library or commands are
+documented in the [CHANGES.md] file.
+
+## CLA
+
+_Contributor Licence Agreement_
+
+This is a legal document that grants certain rights to your contributed
+code.  All non-trivial contributions must be submitted under a CLA.
+Either an ICLA for personal contributions or both an ICLA and a CCLA
```

Comment:
(minor grammar) `Either...` isn't a full sentence, suggest replacing the period/full stop at the end of line 59 with a comma, leading to "All non-trivial contributions must be submitted under a CLA, either an ICLA for personal contributions or both an ICLA and a CCLA for work related contributions."